pkg/config/config_template.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if .Common }}
```

Comment:
nit: does it make sense to have a `Common` flag? :) (by definition I would expect the `Common` part to always be there)

Reply:
You might want to build an executable only running the metadata collection (let's say a standalone tagger) that needs logging but no api_key. Happy to change the name to something more meaningful.